### PR TITLE
Story/DFC-3598 (BE & FE) deconstructed wyd in Sitefinity

### PR DESCRIPTION
**Pull request by trevorkapswarah**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileWhatYouWillDo/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+@if (Model.IsWhatYouWillDoCadView)
+{
+    <section class="job-profile-subsection" id="@Html.Raw(Model.SectionId)">
+        <h2 class="heading-large">@Html.Raw(Model.Title)</h2>
+        @if (Model.IsIntroActive && !string.IsNullOrWhiteSpace(Model.Introduction))
+        {
+            <p>@Model.Introduction</p>
+        }
+        <h3>@Model.DailyTasksSectionTitle</h3>
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileWhatYouWillDo/Index.cshtml]: Are we sure this is correct mark-up? I would have thought this would be a sub section `<section />`?
```
<section id="WhatYouWillDo">
      <h2 class="heading-large">What you will do?</h2>
      <p>intro</p>
      <section class="job-profile-subsection" id="DayToDayTasks">
            <h3>Daily Tasks</h3>
            ....
      </section>
      <section class="job-profile-subsection" id="WorkEnvironment">
            <h3>Work environment</h3>
            ...
      </section>
</section>
```

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileWhatYouWillDo/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿@model DFC.Digital.Web.Sitefinity.JobProfileModule.Mvc.Models.JobProfileWhatYouWillDoViewModel
+
+@if (Model.IsWhatYouWillDoCadView)
+{
+    <section class="job-profile-subsection" id="@Html.Raw(Model.SectionId)">
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Views/JobProfileWhatYouWillDo/Index.cshtml]: Not sure this is a subsection are we using the correct class?

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebServicesConfig.config [hunk 3]
```diff
@@ -20,7 +25,28 @@
 				</add>
 				<add name="jp-integration" urlName="jp-integration">
 					<types>
-						<add urlName="jobprofiles" access="Authenticated" clrType="Telerik.Sitefinity.DynamicTypes.Model.JobProfile.JobProfile" type:this="Telerik.Sitefinity.Web.Api.Configuration.LifecycleTypeSettings, Telerik.Sitefinity.Web.Api" />
+						<add urlName="jobprofiles" access="Anonymous" clrType="Telerik.Sitefinity.DynamicTypes.Model.JobProfile.JobProfile" type:this="Telerik.Sitefinity.Web.Api.Configuration.LifecycleTypeSettings, Telerik.Sitefinity.Web.Api">
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebServicesConfig.config]: are these changes intended?

File: DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj [hunk 4]
```diff
@@ -953,19 +953,20 @@
     <Content Include="AdminApp\assets\manifest.json" />
     <Content Include="OpenAccessNuGet.targets" />
     <Content Include="App_Data\Sitefinity\Export\JobProfile.1.0.25.1.nupkg" />
+    <Content Include="App_Data\Sitefinity\Export\JobProfile.1.0.27.2.nupkg" />
+    <Content Include="Mvc\Views\JobProfileWhatItTakes\DesignerView.Simple.cshtml" />
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj]: Do we need these files under Mvc\Views\JobProfileWhatItTakes in Sitefinity main project?

Comment (trevorkapswarah) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj]: we used to but it seems this later version of Sitefinity allows us to have them co located in the project view folder

Comment (Muthuramana) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj]: So this can be deleted as this new addition for WIT is not needed anymore? Am i missing something? This is not related to WYD anyway isn't it?
